### DISCUSSION TITLE
[Question]: Bulk select events to send to Frigate+ and Bulk verify and save in Frigate+

### DISCUSSION THREAD

**Discussion by anthonws** [blakeblackshear/frigate#11463] [Ask A Question] [answered]
### What is your question:

I'm having a lot of person FPs because of my bicycles handle bars and bottles that I have in my storage room. Is it possible to bulk select images and send to Frigate+ with the same action "This is not a person"?
Also, after submitting the images to Frigate+ and having determined that it is not a person, why do I need to Verify them again in Frigate+ portal? Is there also a way to bulk select images and bulk verify?

**Comment by NickM-27** [comment, marked as answer]
The need to be verified individually. Oftentimes there are other objects in the image. 

The image submission is much improved in the next version of frigate

**Comment by anthonws** [comment]
Hi @NickM-27 , thanks for the reply. I only do "person" detection and don't have use cases for other types of objects (cars, animals, etc.).
But the scenario here is focused on mainly FPs of person, whereas I would just want to say "Not a person".
It would be nice to have this into consideration for the next version :)
I'll be closing this discussion given your feedback. Will wait for the next version.
Thank you once more!
